### DISCUSSION TITLE
Welcome to HelloWord-Keyboard Discussions!

### DISCUSSION THREAD

**Discussion by peng-zhihui** [peng-zhihui/HelloWord-Keyboard#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zgs225** [comment]
There's an English [post](https://kbd.news/Modular-keyboard-1569.html) introduce the project. It should helpful to whom can't read Chinese.

**Comment by AndersonTorres** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hello, I am Anderson Torres, a wannabe computer programmer interested in Nix.

I was/am seeking how to build a custom, ortholinear full (5x20 or 5x25) keyboard. Youtube recommended that video to me, and I liked the project a lot.

> Share ideas.

That being said, my ideas would be:
- ortholinear
- through-hole
- handwire-friendly 
  - especially with [Amoeba single-switch PCB]( https://www.diykeyboards.com/parts/product/amoeba-single-switch-pcbs).
  - and [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/) (risc-v would be better, but it is harder to find...)
